auditbeat/auditbeat.reference.yml

```diff
@@ -250,6 +250,14 @@ auditbeat.modules:
 #       - from: "a"
 #         to: "b"
 #
+# The following example tokenizes the string into fields:
+#
+#processors:
+#- dissect:
+#    tokenizer: "%{key1} - %{key2}"
+#    field: "message"
+#    target_prefix: "extracted"
```

Comment:
We have in the code now `dissect` as default. This should also be adjusted here.